doc/api/toolkits/mplot3d.rst

```diff
@@ -0,0 +1,124 @@
+.. _toolkit_mplot3d-api:
+
+***********
+mplot3d API
+***********
+
+.. contents::
+      :backlinks: none
+
+.. currentmodule:: mpl_toolkits.mplot3d
+
+.. _toolkit_mplot3d-axesapi:
+
+:mod:`~mpl_toolkits.mplot3d.axes3d`
+===================================
+
+.. note::
+   Significant effort went into bringing axes3d to feature-parity with
+   regular axes objects for version 1.1.0. However, more work remains.
+   Please report any functions that do not behave as expected as a bug.
+   In addition, help and patches would be greatly appreciated!
+
+.. autosummary::
+   :toctree: ../_as_gen
+   :template: autosummary.rst
+
+   axes3d.Axes3D
+
+
+.. _toolkit_mplot3d-axisapi:
+
+:mod:`~mpl_toolkits.mplot3d.axis3d`
+===================================
+
+.. note::
+   Historically, axis3d has suffered from having hard-coded constants
+   controlling the look and feel of the 3D plot. This precluded user
+   level adjustments such as label spacing, font colors and panel colors.
```

Comment:
> If you wish to eliminate this note

I don't believe that @story645 wishes to eliminate the note, I think they just want to make sure the information is as clear as possible. For example, the most important bit of information lies at the very end of the note. I'd propose that we move it to the beginning, so something like:

"
See ``mpl_toolkits.mplot3d.axis3d._axinfo`` for a dictionary containing constants that may be modified for controlling the look and feel of mplot3d axes (e.g., label spacing, font colors and panel colors). Historically, axis3d has suffered from having hard-coded constants that precluded user adjustments, and this dictionary was implemented in version 1.1 as a stop-gap measure.
"
I think that pushes the most important information (the name `_axinfo` and what it does) to the front. WDYT @WeatherGod ?